implementations/rust/ockam/ockam_command/src/terminal/mod.rs

```diff
@@ -41,8 +42,8 @@ impl TerminalBackground {
     ///
     /// Reference: https://stackoverflow.com/a/54652367
     pub fn detect_background_color() -> TerminalBackground {
-        match std::env::var("COLORFGBG") {
-            Ok(v) => {
+        match get_env::<String>("COLORFGBG") {
```

Comment:
Here I think we can create a `TerminalColors` struct:
```rust
type Color = u8;

struct TerminalColors {
   foreground: Color,
   background: Color,
}

impl TerminalColors {
    pub fn terminalBackground(&self) -> TerminalBackground {
        if (0..8).contains(&self.backgroundColor) {
            TerminalBackground::Dark
        } else {
            TerminalBackground::Light
        }
    } 
}
```

And provide a `FromString` instance doing the parsing of the 2 colors with `;`